proxy/src/registry.rs

```diff
@@ -10,7 +10,7 @@ use std::convert::TryFrom;
 use std::fmt;
 use std::str::FromStr;
 
-use radicle_registry_client::{self as protocol, ClientT, CryptoPair};
+use radicle_registry_client::{self as protocol, ClientT, CryptoPair, ProjectDomain};
```

Comment:
We only import traits explicitel for the rest we use the `protocol::` qualification.

```suggestion
use radicle_registry_client::{self as protocol, ClientT, CryptoPair};
```